platform/linux-generic/odp_queue.c

```diff
@@ -89,10 +89,10 @@ static int queue_init(queue_entry_t *queue, const char *name,
 	}
 	queue->s.type = queue->s.param.type;
 
-	queue->s.enqueue = queue_enq;
-	queue->s.dequeue = queue_deq;
-	queue->s.enqueue_multi = queue_enq_multi;
-	queue->s.dequeue_multi = queue_deq_multi;
+	queue->s.enqueue = _queue_enq;
+	queue->s.dequeue = _queue_deq;
+	queue->s.enqueue_multi = _queue_enq_multi;
+	queue->s.dequeue_multi = _queue_deq_multi;
```

Comment:
@nagarahalli Yes, but I'd second @heyi-linaro proposal to leave old `queue_enq/_dec/_enq_multi/_deq_multi` names as is and instead rename `odp_queue_enq` to `internal_queue_enq` or something like that.